core/src/main/resources/i18n/displayStrings.properties

```
@@ -3024,7 +3024,7 @@ You must empty this wallet before attempting to restore an older one, as mixing
 together can lead to invalidated backups.\n\n\
 Please finalize your trades, close all your open offers and go to the Funds section to withdraw your bitcoin.\n\
 In case you cannot access your bitcoin you can use the emergency tool to empty the wallet.\n\
-To open that emergency tool press \"alt + e\" or \"option + e\" .
+To open that emergency tool press \"Alt + e\" or \"Ctrl + e\" or \"Cmd + e\" .
```

```suggestion
To open the emergency tool press \"Alt+e\" or \"Cmd/Ctrl+e\".
```
To be consistent with the way key shortcuts have been written elsewhere (not entirely consistent yet, but would like to make it that way).

---

Ah yes good catch!